pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/globalcomputeforwardingruletcp/_http.log

```diff
@@ -722,7 +809,7 @@ User-Agent: kcc/controller-manager
 x-goog-request-params: project=${projectId}
 
 {
-  "IPAddress": "8.8.8.8",
+  "IPAddress": "34.54.63.45",
```

Comment:
That's expected because we are using fake ip address in mock ComputeAddress server